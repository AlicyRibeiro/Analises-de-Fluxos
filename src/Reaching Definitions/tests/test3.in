3 7
a = b + c
b = a + d
a = a + b
x = y + z
y = x + 1
z = y + 2
x = z + 3
1 2
2
0
