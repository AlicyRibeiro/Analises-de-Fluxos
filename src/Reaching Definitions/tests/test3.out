Bloco 0:
  IN : { }
  OUT: { (a,0,0) (b,0,1) }
Bloco 1:
  IN : { (a,0,0) (b,0,1) }
  OUT: { (a,0,0) (b,0,1) (x,1,0) (y,1,1) }
Bloco 2:
  IN : { (a,0,0) (b,0,1) (x,1,0) (y,1,1) }
  OUT: { (a,0,0) (b,0,1) (x,2,1) (y,1,1) (z,2,0) }
